### PR TITLE
[最新面試心得] 顯示 originalCompanyName

### LINKED ISSUE #1342: [公司名稱合併] 首頁 experience 卡片顯示 canonical company name
## Expected Result
首頁的 experience 卡片，要顯示 original company name
我印象我們討論出來的結論是，使用者留的資訊會保留，會用 original company name，我們自己 aggregate 的資料才會用 canonical company name

## Actual Result
首頁的 experience 卡片，顯示的是 canonical company name
Screenshot 2024-06-19 at 7 08 06 PM

不過這個問題並不是特別嚴重，不影響公司合併上線

### PR DESCRIPTION
**Pull request by peteranny**
Close #1342  

## 這個 PR 是？ 

`popular_experience` 取 `originalCompanyName`
加上 prop types

## Screenshots  

畫面擷取於 2024-06-19 23 30 20

## 我應該如何手動測試？ 

- [ ] http://localhost:3000/ 首頁檢查最新面試與心得的卡片顯示的是 originalCompanyName

### CODE REVIEW

File: src/components/LandingPage/ExperienceBlock/index.js [hunk 1]
```diff
@@ -51,9 +51,18 @@ const ExperienceBlock = ({ data, size, backable }) => {
 };
 
 ExperienceBlock.propTypes = {
-  backable: PropTypes.bool,
-  data: PropTypes.object.isRequired,
-  size: PropTypes.oneOf(['s', 'm', 'l']),
+  backable: PropTypes.bool.isRequired,
+  data: PropTypes.shape({
+    _id: PropTypes.string.isRequired,
```
Comment (mark86092) [inline src/components/LandingPage/ExperienceBlock/index.js:56]: it looks like `_id` is mapping from `id` which comes from api.

maybe we can "correct" it to `id`

**Review by mark86092 — APPROVED** [review]

File: src/components/LandingPage/ExperienceBlock/index.js (hunk 1 above)
Comment (peteranny) [reply src/components/LandingPage/ExperienceBlock/index.js]: Updated

**Comment by peteranny** [comment]
移除一些 old migrations

**Review by mark86092 — APPROVED** [review]
LGTM

**Pull request merged**